arc/cli/utils/db_connection.py

```
@@ -0,0 +1,91 @@
+        self._client: Optional[ArcDBClient] = None
+        self._is_connected = False
+        self._connection_url = os.environ.get("TIMESCALE_SERVICE_URL")
+        
+    @property
+    def is_available(self) -> bool:
+        """Check if database connection is available."""
+        return self._is_connected
+    
+    async def initialize(self) -> bool:
+        """Initialize database connection with retry logic.
+        
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)